src/SlackThreads.jl

```diff
@@ -111,8 +112,16 @@ Valid `object`s are:
 * Emits `@debug` logs when sending requests and recieving responses from the Slack API.
 * Emits `@warn` logs with the contents of requests to the Slack API when the channel or token is not configured correctly (in lieu of sending a request)
 * Emits `@error` logs when an exception is encountered or Slack returns an error response.
+
+## Message splitting
+
+If there are many attachments, the messages will be split into multiple messages. By default,
+[`SlackThreads.format_message_counts`](@ref) is used to format text to indicate when a message has been split.
+One may pass a function with the same signature to customize this text, or e.g. pass `(i, n) -> ""` to not
+display any text indicating that the messages have been split.
 """
-function (thread::SlackThread)(text::AbstractString, uploads...)
+function (thread::SlackThread)(text::AbstractString, uploads...;
+                               format_message_counts=format_message_counts)
```

Comment:
Should the `combine_texts` function be passed in instead? Otherwise you cannot control the max length used

Reply:
True, but I think that's fine. Is there any reason a user would want to customize it?

Reply:
You're right, it's just more flexbility, and maybe they want to do a different splitting thing altogether. Changed.